src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -0,0 +1,157 @@
+import type { Router as RemixRouter } from '@remix-run/router';
+import { screen, waitFor } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+
+import { CREATE_BLUEPRINT, EDIT_BLUEPRINT } from '../../../../../constants';
+import { mockBlueprintIds } from '../../../../fixtures/blueprints';
+import { usersCreateBlueprintRequest } from '../../../../fixtures/editMode';
+import {
+  blueprintRequest,
+  clickBack,
+  clickNext,
+  enterBlueprintName,
+  getNextButton,
+  interceptBlueprintRequest,
+  interceptEditBlueprintRequest,
+  openAndDismissSaveAndBuildModal,
+  renderEditMode,
+  verifyCancelButton,
+} from '../../wizardTestUtils';
+import {
+  clickRegisterLater,
+  goToRegistrationStep,
+  renderCreateMode,
+} from '../../wizardTestUtils';
+
+let router: RemixRouter | undefined = undefined;
+
+const goToUsersStep = async () => {
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+};
+
+const goToReviewStep = async () => {
+  await clickNext(); // Timezone
+  await clickNext(); // Locale
+  await clickNext(); // Hostname
+  await clickNext(); // Kernel
+  await clickNext(); // First boot
+  await clickNext(); // Details
+  await enterBlueprintName();
+  await clickNext(); // Review
+};
+
+const addValidUser = async () => {
+  const user = userEvent.setup();
+  const addUser = await screen.findByRole('button', { name: /add a user/i });
+  expect(addUser).toBeEnabled();
+  await waitFor(() => user.click(addUser));
+  const enterUserName = screen.getByRole('textbox', {
+    name: /blueprint user name/i,
+  });
+  const nextButton = await getNextButton();
+  await waitFor(() => user.type(enterUserName, 'best'));
+  await waitFor(() => expect(enterUserName).toHaveValue('best'));
+  await waitFor(() => expect(nextButton).toBeEnabled());
+};
+
+describe('Step Users', () => {
+  beforeEach(async () => {
+    vi.clearAllMocks();
+    router = undefined;
+  });
+
+  test('without adding user loads timezone', async () => {
+    const user = userEvent.setup();
+    await renderCreateMode();
+    await goToRegistrationStep();
+    await clickRegisterLater();
+    await goToUsersStep();
+    const addUser = await screen.findByRole('button', { name: /add a user/i });
```

Comment:
One goal for these tests is to limit their scope to test one specific thing and avoid duplicate test coverage to the extent possible. This will hopefully make them easier to manage and work with.

This test makes sure that the "next" button works and that the appropriate step loads when it is clicked. That's a good thing to test, and we have similar tests in all the other steps.

However, do we really need to check and make sure that the add user button is enabled? I think that the test where we actually add a user already covers that case.